include/field2d.hxx

```diff
@@ -302,29 +302,31 @@ const Field2D abs(const Field2D &f);
 
 /*!
  * Calculates the minimum of a field, excluding
- * the boundary/guard cells. 
+ * the boundary/guard cells by default.
  * By default this is only on the local processor,
  * but setting allpe=true does a collective Allreduce
  * over all processors.
  *
  * @param[in] f  The field to loop over
  * @param[in] allpe  Minimum over all processors?
+ * @param[in] rgn  the boundaries that should be ignored
```

Comment:
I think this should be "rgn  Calculate the minimum inside this region".

Currently it sounds like `rgn` should contain the points you want to ignore, rather than the points you want to consider

Reply:
rgn can only be used to ignore the boundaries - no general region.
maybe
```
rgn  The boundaries to be included in the calculation
```
?

Reply:
What about `RGN_ALL`? We're also moving towards supporting very general regions which may even be some subset of the interior points, so this should be

    * @param[in] rgn  The region to calculate the minimum over

and likewise for `max`

Reply:
`RGN_ALL` means to include all boundaries - so matches the description as far as I understand.

I wrote the documentation for what is currently done - not what will be possible in the future.
I can change of course - but I am not the expert on the generalized region thing, am not sure whether or when it is going to get merged, when that function is adapted, so I didn't match it.